### PR TITLE
SEO-172157 ASP.NET Core Image Alt

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @gowthamanthavasiyappan ,

Have changed image alt and renamed.

Regards,
Yvone.

### CODE REVIEW

File: aspnet-core/Chart/Appearance.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The chart displays different series in different colors, by default. You can cus
 {% endhighlight %}
 
 
-![](Appearance_images/Appearance_img1.png)
+![Custom color palette in ASP.NET Core Chart](appearance_images/aspnetcore-chart-custom-color-palette.png)
```
Comment (gowthamanthavasiyappan) [inline aspnet-core/Chart/Appearance.md:27]: Have you changed the folder name also? check all places.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: aspnet-core/Chart/Appearance.md (hunk 1 above)
Comment (Yvone-Atieno) [reply aspnet-core/Chart/Appearance.md:27]: Yes
image

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**